app/src/main/java/com/duckduckgo/app/feedback/ui/FeedbackViewModel.kt

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2018 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.feedback.ui
+
+import android.arch.lifecycle.MutableLiveData
+import android.arch.lifecycle.ViewModel
+import com.duckduckgo.app.feedback.api.FeedbackSender
+import com.duckduckgo.app.global.SingleLiveEvent
+
+
+class FeedbackViewModel(private val feedbackSender: FeedbackSender) : ViewModel() {
+
+    data class ViewState(
+        val isBrokenSite: Boolean = false,
+        val url: String? = null,
+        val showUrl: Boolean = false,
+        val message: String? = null,
+        val submitAllowed: Boolean = false
+    )
+
+    sealed class Command {
+        object FocusUrl : Command()
+        object FocusMessage : Command()
+        object Finish : Command()
+    }
+
+    val viewState: MutableLiveData<ViewState> = MutableLiveData()
+    private val viewValue: ViewState get() = viewState.value!!
```

Comment:
This is a personal thing, but I tend to group variables with similar access modifiers together, with private nearer to the constructor, if there is one. I don't know if there's a convention for this in Kotlin though?